pacman/utilities/utility_calls.py

```diff
@@ -63,45 +49,6 @@ def _is_constraint_supported(constraint, supported_constraints):
                for supported_constraint in supported_constraints)
 
 
-def check_algorithm_can_support_constraints(
```

Comment:
I think if this is being deleted then _is_constraint_supported (above it) can go as well.